src/wp-admin/includes/update.php

```diff
@@ -306,7 +306,7 @@ function update_nag() {
 	if ( current_user_can( 'update_core' ) ) {
 		$msg = sprintf(
 			/* translators: 1: URL to WordPress release notes, 2: New WordPress version, 3: URL to network admin, 4: Accessibility text. */
-			__( '<a href="%1$s">WordPress %2$s</a> is available! <a href="%3$s" aria-label="%4$s">Please update now</a>.' ),
+			__( '<a href="%1$s" target="_blank" rel="noreferrer noopener">WordPress %2$s</a> is available! <a href="%3$s" aria-label="%4$s">Please update now</a>.' ),
```

Comment:
FWIW `target="_blank"` implies `rel="noopener"` behavior in all major browsers these days.

Also `noreferrer` is not needed at all. WordPress doesn't use it anymore. See https://core.trac.wordpress.org/changeset/49215

Reply:
Another ticket that targets removing `noopener` throughout Core: https://core.trac.wordpress.org/ticket/53843 - so this should definitely be removed from both links the PR.

Reply:
Yeah it's not needed.